assets/js/phoenix.js

```diff
@@ -911,19 +923,16 @@ export class Socket {
   /**
    * @param {Object} data
    */
-  push(data){
-    let {topic, event, payload, ref, join_ref} = data
-    let callback = () => {
-      this.encode(data, result => {
-        this.conn.send(result)
-      })
+  push(data, reliable = true){
```

Comment:
I think the `reliable` parameter is left from your earlier version and isn't used anymore.